frontend/public/components/modals/expand-pvc-modal.tsx

```diff
@@ -70,15 +67,32 @@ const ExpandPVCModal = withHandlePromise((props: ExpandPVCModalProps) => {
           </Trans>
         </p>
         <label className="control-label co-required">{t('modal~Size')}</label>
-        <RequestSizeInput
```

Comment:
@spadgett RequestSizeInput contains a input field which we really don't need! We just need the dropdown! So I made it this way! I would like to know you point on this?

Thanks

Reply:
Hi, I disagree. You do need to two fields: the number input and the unit dropdown. That should be managed by the `RequestSizeInput` component. Whether we have a number spinner or could be a prop. This is important so we're not duplicating logic and have consistency everywhere we show number + unit in a form. If we ever change how we handle that, this instance would have to be updated separately otherwise.

I personally feel that the number spinner is a bigger change that should be handled as a separate RFE so we can do it consistently across all the places we use this input. We can also look at updating to the Patternfly component which looks nicer. I think it would be better to simply update the text to fix the usability problem and then do the spinner as a follow on.

@yuvalgalanti @bipuladh Opinion?

Reply:
Agreed. We could move the `NumberSpinner` to `InputGroup` PF component. And `RequestSizeInput` could internally use `NumberSpinner` or we could just change the `input` element inside `RequestSizeInput` to use `InputGroup`.
Around 13 components use `RequestSizeInput` so IMO it is safe to make change to `RequestSizeInput` in this PR itself. 

Regarding creating a different RFE and  moving out the increment/decrement buttons, I would defer that decision to @yuvalgalanti since QE will verify this bug based on the UX design. So we will need comments from @yuvalgalanti and @spadgett on the BZ regarding the change in our approach.